src/main/groovy/org/starchartlabs/flare/publishing/task/ContainerBuildTask.groovy

```diff
@@ -59,7 +59,7 @@ public class ContainerBuildTask extends Exec {
             container.getLabels().entrySet()
                     .forEach{e ->
                         currentArgs.add("--label")
-                        currentArgs.add("\"${e.key}=${e.value}\"")
```

Comment:
@DMaxfield-BDS  What about switching to `currentArgs.add('"' + "${e.key}=${e.value}" + '"')`? That might allow this to work on both Unix and Windows. I have thus far not been able to get a working Unix environment to try this - can you update to that and see how it behaves? I can then verify Windows again

Reply:
Updated. Output of `docker inspect` from code change:

```
            "Labels": {
                "label 3": "three 3",
                "label1": "one",
                "label2": "two"
            }
```